tools/SourceKit/tools/sourcekitd-test/TestOptions.cpp

```diff
@@ -290,5 +294,11 @@ bool TestOptions::parseArgs(llvm::ArrayRef<const char *> Args) {
     return true;
   }
 
+  if (Timings) {
+    PrintRequest = false;
+    PrintResponseAsJSON = false;
+    PrintRawResponse = false;
+  }
```

Comment:
I think I'm going to pull the -timings stuff from this patch to give it a little more attention later, it's not really necessary for the functionality right now and this is already overcooked.